engines/scumm/he/logic/funshop.cpp

```diff
@@ -67,7 +67,7 @@ int32 LogicHEfunshop::dispatch(int op, int numArgs, int32 *args) {
 void LogicHEfunshop::op_1004(int32 *args) {
 	double data[8], at, sq;
 	int32 x, y;
-	int i=0;
+	int i;
```

Comment:
`i = 0` instead of `i=0`

Reply:
This gets reassigned immediately, so there's no reason for that assignment

Reply:
...Sorry, I read green and red the opposite way. I swear I'm not daltonic 😛